topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.md

```diff
@@ -0,0 +1,198 @@
+
+# Introduction
+{:.no_toc}
+
+RNA-Seq analysis helps researchers finding new genes and spliced variants, and provides a quantification of cell- and context- specific gene expression. Its data is however complex to handle, and requires both computer science and mathematical knowledge to be managed and interpreted.
+
+Visualization techniques are key to overcome the complexity of RNA-Seq data, and represent valuable tools to gather information and insights.
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Reasons for visualizing RNA-Seq results
+
+To make sense of the available RNA-Seq data, and overview the condition-specific gene expression levels of the provided samples, we need to visualize our results. Here we will use [CummeRbund](http://compbio.mit.edu/cummeRbund/).
+
+CummeRbund is an open-source tool that simplifies the analysis of a CuffDiff RNA-Seq output. In particular, it helps researchers:
+- managing, integrating, and visualizing the data produced by CuffDiff
+- simplifying data exploration
+- providing a bird's-eye view of the expression analysis by describing relationships betweeen genes, transcripts, transcription start sites, and CDS regions
+- exploring subfeatures of individual genes or gene-sets
+- creating publication-ready plots
+
+A typical workflow for the visualization of RNA-Seq data involving CummeRbund:
+
+![workflow](../../images/cummerbund-rna-seq-viz-with-cummerbund.png)
+
+CummeRbund reads your RNA-Seq results from a [SQLite](https://www.sqlite.org/) database. This database has to be created using CuffDiff's SQLite output option.
+
+> ### {% icon tip %} Tip: SQLite output with CuffDiff
+>
+> Instruct CuffDiff to organize its output in a SQLite database for later read with CummeRbund
+>
+> ![SQLite output](../../images/cummerbund-cuffdiff-set-sqlite.png)
+{: .tip}
+
+# Importing RNA-Seq result data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history
+> 2. Import the [`CuffDiff SQLite`](https://zenodo.org/record/1001880/files/CuffDiff_SQLite_database.sqlite) dataset
+>
+>    > ### {% icon tip %} Tip: Importing data via links
+>    >
+>    > * Copy the link location
+>    > * Open the Galaxy Upload Manager
+>    > * Select **Paste/Fetch Data**
+>    > * Paste the link into the text field
+>    > * Press **Start**
+>    {: .tip}
+>
+>    > ### {% icon comment %} Comments
+>    >
+>    > Rename the dataset to "RNA-Seq SQLite result data"
+>    {: .comment}
+> By default, when data is imported via its link, Galaxy names it with its URL.
+{: .hands_on}
+
+CuffDiff's output data is organized in a SQLite database, so we need to extract it to be able to see how it looks like.
+
+For this tutorial, we are interested in CuffDiff's tested transcripts for differential expression.
+
+> ### {% icon hands_on %} Hands-on: Extract CuffDiff results
+>
+> 1. **Extract CuffDiff** {% icon tool %}: Lookup for this tool in the search bar, and select it
```

Comment:
this tool is from the `cummerbund_to_tabular` toolshed repo correct? could you add this to the tools.yaml file as well?

Reply:
Thanks @shiltemann. It slipped my mind. Added